code/game/verbs/who.dm

```diff
@@ -1,63 +1,98 @@
-
 /client/verb/who()
 	set name = "Who"
 	set category = "OOC"
 
-	var/msg = "<b>Current Players:</b>\n"
+	var/list/lines = list()
+	var/list/temp = list()
+
+	for(var/client/C in GLOB.clients)
+		if(C.holder && C.holder.big_brother) // BB doesn't show up at all
+			continue
+
+		if(C.holder && C.holder.fakekey)
+			temp.Add(C.holder.fakekey)
+		else
+			temp.Add(C.key)
```

Comment:
`+=` is faster than `Add()` for lists. Same for below.
```suggestion
			temp += C.holder.fakekey
		else
			temp += C.key
```

Reply:
Will change